tests/unit/ProductFeed/Integrations/OpenAi/ProductMapperTest.php

```
@@ -1042,4 +1046,135 @@ public function test_map_product_category_single_level(): void {
+			);
+		}
+
+		$product = WC_Helper_Product::create_simple_product();
+		$result  = $this->sut->map_product( $product );
+
+		$this->assertArrayHasKey( 'shipping', $result );
+		$shipping = $result['shipping'];
+		$this->assertEquals( $expected_string, $shipping );
+	}
+
+	private function create_shipping_zone( string $name, string $location_type, string $location_code, $shipping_methods = [], $id = null ): WC_Shipping_Zone {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$name'. (undefined)

(UnusedFormalParameter)